data/json/items/chemicals_and_resources.json

```diff
@@ -1687,5 +1687,62 @@
     "price_postapoc": 20000,
     "count": 1,
     "stack_size": 1
+  },
+  {
+    "id": "red_phosphorous",
+    "type": "AMMO",
+    "ammo_type": "components",
+    "category": "chems",
+    "name": { "str_sp": "red phosporous" },
+    "description": "An allotrope of the element phosporous. It's what you use to light matches, but can also be used in the synthesis of methamphetamine.",
+    "weight": "1800 mg",
+    "volume": "100 ml",
+    "//": "Density around 2.2g/cm3, but since it's a powder, it's a bit lower, around 1.8 g/cm3",
+    "container": "box_small",
+    "symbol": "=",
+    "color": "red",
+    "price": 50,
+    "price_postapoc": 2,
+    "count": 100
+  },
+  {
+    "id": "acetic_anhydride",
+    "type": "COMESTIBLE",
+    "category": "chems",
+    "name": { "str": "acetic anhydride" },
```

Comment:
```suggestion
    "name": { "str_sp": "acetic anhydride" },
```

Reply:
I'm accepting suggestion, but I don't know what's the difference between str and str_sp

Reply:
Im accepting the changes but I don't know the difference between the two strings

Reply:
`str_sp` makes the string to have the same plural form as the singular one.